docs/reference/node/telemetry-config.md

```diff
@@ -0,0 +1,31 @@
+title: Telemetry
```

Comment:
I'd recommend using `command-name` format when referring to commands or command line arguments.